workflow-templates/crowdin-sync.properties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Crowdin sync",
+  "description": "For syncing sources and translations with Crowdin.",
+  "iconName": "crowdin",
+  "filePatterns": [
+      "^crowdin.ya?ml$"
```

Comment:
isn't it the a who is? instead of the m?

Reply:
```suggestion
      "^crowdin.y?aml$"
```

Reply:
Ah or ? applies on the letter after actually, not before 😄

Reply:
I just copied from `workflow-templates/crowdin-download.properties.json`, I actually don't know what this option does

Reply:
Anyway looks like a regex, and regex ! and ? characters modify the preceding character, right?